src/LiveComponent/assets/src/morphdom.ts

```diff
@@ -1,5 +1,5 @@
 import { cloneHTMLElement, setValueOnElement } from './dom_utils';
-import morphdom from 'morphdom';
+import 'idiomorph';
```

Comment:
This looks funny. Is `iodiomorph` setting a global `Idiomorph` variable?

Also, we should tweak the packaging system (rollup) so that `idiomorph` is NOT packaged with our final `live_controller.js` but is, instead, left as its import (this might be as simple as moving it from `dependencies` to `peerDependencies` but I can't remember for sure. That will allow users to install & reuse `idiomorph` across LiveComponents & Turbo vs having a different copy for each one. We'll then also need to add `idiomorph` to the `importmap` part of `package.json` so it's added to `importmap.php` on install.